src/common/thrift/ThriftClientManager-inl.h

```diff
@@ -84,7 +84,7 @@ std::shared_ptr<ClientType> ThriftClientManager<ClientType>::client(const HostAd
     }
   });
   auto clientChannel = apache::thrift::RocketClientChannel::newChannel(std::move(socket));
-  if (timeout > 0) {
+  if (timeout >= 0) {
```

Comment:
`timeout` is a uint.

Reply:
It is supposed to, I'll remove the condition then.